kieserver/modules/kieserver/module.yaml

```diff
@@ -36,6 +36,10 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "kie-server-services-jbpm-cluster-7.x.Final.jar"
```

Comment:
I don't think it a good idea have 7.x here, in this case, I would prefer to have the full and exact version here.
@jakubschwan @AntStephenson wdyt?

Reply:
@AntStephenson also, will this jar be available on brew?

Reply:
I could change and use a placeholder, but how we retrieve a specific version and its md5 without add here ?

Reply:
I agree we should probably use the full exact version here. Also, IIUC the artifact will be in Indy and it can be fetched from there. CEKit constructs a yaml file instructing OSBS where to fetch the artifacts from.